genai/README.md

```diff
@@ -4,7 +4,8 @@ In this lab, we will go through LLM Chatbot AMP using Cloudera Machine Learning
 
 1. On login, you will be taken to Trials Home Page
 2. Click, **Start Trial** go to your Machine Learning Workspace
-3. [Here](https://app.getreprise.com/launch/MXxjDe6/) is the Reprise demo with step by step tutorial for the LLM Chatbot
+3. [Here](https://app.getreprise.com/launch/MXxjDe6/) is the Reprise demo with step by step tutorial for the LLM Chatbot 
+4.  [Here](https://app.getreprise.com/launch/dyR7W26/) is the Reprise demo showcasing Cloudera AI Inference capabilities.
```

Comment:
Why are we adding this here? This page talks about the first reprise. What's the flow users go through?